src/Core/Services/TypeHelper.cs

```diff
@@ -16,6 +17,10 @@ public static class TypeHelper
     {
         /// <summary>
         /// Maps .NET Framework types to DbType enum
+        /// Not Adding this hard mapping for System.DateTime/DateTimeOffset,
+        /// since System.DateTime is used for both SqldbType.DateTime and SqldbType.DateTime2.
+        /// So no change required here as separate logic exist which does the mapping
```

Comment:
nit: the comment has no context from the PR Comment thread. Mapping of `System.DateTime`, `DateTimeOffset`, 
and `System.DateTime` to what? 

Also SqlDbType is not relevant in this mapping dictionary is it?

Reply:
yep, not SqlDbType but DbType